test/unit/test.ts

```diff
@@ -0,0 +1,26 @@
+/*
+	File: test/unit/unit-test.ts
+	cpuabuse.com
+*/
+
+/**
+ * Performs a series of unit tests.
+ */
+
+import { testShowNonPrinting } from "./pipeline/show-nonprinting";
+import { testShowTabs } from "./pipeline/show-tabs";
+import { testShowEnds } from "./pipeline/show-ends";

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing file extension for "./pipeline/show-ends"](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5042286)